content/applications/finance/payment_acquirers/stripe.rst

```diff
@@ -61,9 +54,10 @@ Odoo.sh or On-premise
 .. _stripe/api_keys:
 
 Publishable and Secret keys
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
+---------------------------
 
-Odoo needs your **API Credentials** to connect with your Stripe account, which comprises:
+If Odoo still needs your **API Credentials** to connect with your Stripe account, these are the
```

Comment:
I understand what this needs to change but I'm not sure about the formulation... It's a bit weird for me to say "if Odoo needs something from you". Could we maybe change it to "If you were asked to fill your credentials in", as a reference to the instructions above?

@LoredanaLrpz What do you think?

Reply:
I'm not crazy about "if you were asked" either, 
what do you think about "In case your API Credentials are still required to connect with your Stripe account, "

Reply:
:+1: I'd just get rid of the "still" because this is the 'normal' flow when you're on-premise/on Odoo.sh :)